### PR TITLE
Add to community resources - MatBlazor and BlazorSplit

### PR DESCRIPTION
**Pull request by SamProf**
I think MatBlazor currently bigget Material Design components library and fast growing.
Thank you.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [BlazorNodaTimeDateTimePicker](https://github.com/nheath99/BlazorNodaTimeDateTimePicker) &ndash; A date picker using NodaTime.
 * [Blazorous](https://github.com/chanan/Blazorous) &ndash; Maintainable CSS with Blazor.
 * [BlazorSignalR](https://github.com/csnewman/BlazorSignalR) &ndash; ASP.NET Core SignalR Core .NET client support for Blazor.
+* [BlazorSplit](https://github.com/BlazorComponents/BlazorSplit) ([Demo](https://blazorcomponents.github.io/BlazorSplit/)) ([nuget.org](https://www.nuget.org/packages/BlazorSplit)) &ndash; Resizeable split views.
```
Comment (danroth27) [inline community.md]: We don't typically link to the NuGet package here. We've been linking primarily to the GitHub repo, and then optionally to a live demo site.

File: community.md [hunk 2]
```diff
@@ -35,6 +36,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [ChartJs.Blazor](https://github.com/mariusmuntean/ChartJs.Blazor) &ndash; Blazor Component that wraps ChartJS.
 * [Blazor Extensions Home](https://github.com/BlazorExtensions/Home) &ndash; Home for Blazor Extensions.
 * [Logging](https://github.com/BlazorExtensions/Logging) &ndash; Microsoft Extension Logging implementation for Blazor.
+* [MatBlazor](https://github.com/BlazorComponents/MatBlazor) ([Demo](https://blazorcomponents.github.io/MatBlazor/)) ([nuget.org](https://www.nuget.org/packages/MatBlazor)) &ndash; Google Material components for web in Blazor.
```
Comment (danroth27) [inline community.md]: We don't typically link to the NuGet package here. We've been linking primarily to the GitHub repo, and then optionally to a live demo site.

File: community.md (hunk 1 above)
Comment (SamProf) [reply community.md]: I have removed nuget packages!

File: community.md (hunk 2 above)
Comment (SamProf) [reply community.md]: I have removed nuget packages!

Comment (SamProf) [reply community.md]: nuget packages removed!

**Pull request merged**